xwiki-platform-core/xwiki-platform-tree/xwiki-platform-tree-webjar/src/main/webjar/finder.css

```diff
@@ -15,7 +15,7 @@ input[type="text"].xtree-finder.loading {
 div.suggestItems.xtree-finder-suggestions {
   background-clip: padding-box;
   border: 1px solid rgba(0, 0, 0, 0.15);
-  // Note: we can't replace the hardcoded radius with a variable here as the file is compiled to css at build time. 
+  /* Note: we can't replace the hardcoded radius with a variable here as the file is compiled to css at build time.*/
```

Comment:
The content of this comment seems wrong, this is not compiled anymore from what I understand. Further, there shouldn't speak anything against actually using a variable now that we have CSS variables.